plugins/module_utils/network/sonic/facts/ospfv3_interfaces/ospfv3_interfaces.py

```diff
@@ -0,0 +1,149 @@
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+from copy import deepcopy
+
+from ansible_collections.ansible.netcommon.plugins.module_utils.network.common import (
+    utils,
+)
+from ansible_collections.dellemc.enterprise_sonic.plugins.module_utils.network.sonic.argspec.ospfv3_interfaces.ospfv3_interfaces import Ospfv3_interfacesArgs
+
+from ansible_collections.dellemc.enterprise_sonic.plugins.module_utils.network.sonic.sonic import (
+    to_request,
+    edit_config
+)
+from ansible_collections.dellemc.enterprise_sonic.plugins.module_utils.network.sonic.utils.utils import (
+    remove_empties_from_list
+)
+from ansible.module_utils.connection import ConnectionError
+
+
+class Ospfv3_interfacesFacts(object):
+    """ The sonic ospfv3_interfaces fact class
+    """
+
+    def __init__(self, module, subspec='config', options='options'):
+        self._module = module
+        self.argument_spec = Ospfv3_interfacesArgs.argument_spec
+        spec = deepcopy(self.argument_spec)
+        if subspec:
+            if options:
+                facts_argument_spec = spec[subspec][options]
+            else:
+                facts_argument_spec = spec[subspec]
+        else:
+            facts_argument_spec = spec
+
+        self.generated_spec = utils.generate_dict(facts_argument_spec)
+
+    def populate_facts(self, connection, ansible_facts, data=None):
+        """ Populate the facts for ospfv3_interfaces
+        :param connection: the device connection
+        :param ansible_facts: Facts dictionary
+        :param data: previously collected conf
+        :rtype: dictionary
+        :returns: facts
+        """
+        objs = []
+
+        all_ospfv3_interface_configs = {}
+        if not data:
+            all_ospfv3_interface_configs = self.get_ospfv3_interfaces()
+
+        for ospfv3_interface_config in all_ospfv3_interface_configs:
+            if ospfv3_interface_config:
+                obj = self.render_config(self.generated_spec, ospfv3_interface_config)
+                if obj:
+                    objs.append(obj)
+
+        ansible_facts['ansible_network_resources'].pop('ospfv3_interfaces', None)
+        facts = {}
+        if objs:
+            params = utils.validate_config(self.argument_spec, {'config': objs})
+            facts['ospfv3_interfaces'] = remove_empties_from_list(params['config'])
+
+        ansible_facts['ansible_network_resources'].update(facts)
+        return ansible_facts
+
+    def render_config(self, spec, conf):
```

Comment:
This function returns same config.
Hence remove this and also change line 56 to 58 as objs.append(ospfv3_interface_config)